src/styles/globals.css

```
@@ -1,12 +1,52 @@
+  --color-green-300: #16a34a;
+
+  /* Single colors */
+  --color-orange: #f97316;
+  --color-yellow: #eab308;
+  --color-purple: #a855f7;
+  --color-white: #ffffff;
+  --color-black: #000000;
+}
 
 html {
+  font-family: Pretendard;
```

Biome (2.1.2)

[error] Generic font family missing.

Consider adding a generic font family as a fallback.
For examples and more information, see the MDN Web Docs


(lint/a11y/useGenericFontNames)